packages/dart/noports_core/lib/src/npa/npa_params.dart

```diff
@@ -24,7 +27,7 @@ class NPAParams {
     // Arg check
     ArgResults r = parser.parse(args);
 
-    String authorizerAtsign = r['atsign'];
+    String authorizerAtsign = (r['atsign'] as String).toAtsign();
```

Comment:
The toAtsign() extension I added in at_utils can throw errors, as it is functionally equivalent to the fixAtsign util from before.

Reply:
Thanks Xavier. The call to `NPAParams.fromArgs` is wrapped in its own try-catch block in npp_atserver's `main` function